libs/core-data/src/lib/repositories/repositories.graphql.ts

```diff
@@ -0,0 +1,37 @@
+import gql from 'graphql-tag';
+
+const repositoryFragment = gql`
+  fragment repositoryFragment on Repository {
+    id
+    name
+    description
+    homepageUrl
+    url
+    forkCount
+    isPrivate
+    isFork
+    languages(first: 1) {
+      nodes {
+        name
+        color
+        id
+      }
+    }
+    createdAt
+    updatedAt
+    pushedAt
+  }
+`;
+
+export const repositoriesQuery = gql`
+  query repositoriesQuery($login: String!) {
```

Comment:
So here, we get the users' repositories. How will we account for organization repositories?

Reply:
I wasn't going to add that till after the user repo was fleshed out and running